examples/ant-design/app/root.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration
+} from "remix";
+import type { MetaFunction } from "remix";
+
+//imported the ant design styles
+import 'antd/dist/antd.css'
+import './styles/index.css'
```

Comment:
This won't get those styles on the page. You have to add a `links` export: https://remix.run/docs/en/v1/guides/styling#regular-stylesheets